### ISSUE TITLE
Mochaloader not available for import until after gulp test is run

### ISSUE THREAD

**Issue by syo** [FullScreenShenanigans/gulp-shenanigans#56]
Currently the mochaloader files are generated the first time "gulp test" is run. This leads to some confusion, as the unit tests will have errors from being unable to import it, and is mildly unintuitive.

The problem is resolved as long as gulp test is run.

**Comment by JoshuaKGoldberg** [comment]
Yeah, it's a confusing part of the design. A couple of things that could make it better:
* Run `test:html` and `gulp test:utilities` as part of project setup
* Document this behavior
